src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -214,8 +214,116 @@ As AWS is a paying service, when the Scheduler is stopped normally (without remo
+ This tag is valued with this configuration option and the Node Source name. Concurrent ProActive scheduler can therefore operate concurrent clusters in the same AWS regions, with the same node source name, provided that they diverge on the affectied to this parameter.
+ This option is mandatory.
+
+	- *awsKey:* This field must be filled with the content of the AWS key from the administrator.
+ This option is mandatory.
+
+	- *awsSecretKey:* The administrator must complete this field with the content of its secret access key.
+ This option is mandatory.
+
+	- *maxVms:* This parameter defines the number of maximum tolerated instances on the infrastructure: the instance allocations will be systematically blocked if the Resource Manager tries to overpass this threshold. 
+ This option is mandatory and cannot exceed 100.
+
+	- *defaultInstanceType:* This parameter defines the instance type to use for AWS instance operating ProActive. This parameter should choosen accordingly to the expected processing to be performed on the node source. 
+ This option is mandatory and has to be filled after one AWS InstanceType name (e.g. t3.large).
+
+	- *amiId:* The administrator defines in this field the ID of the AMI to use to bootstrap instance operating ProActive nodes. 
+ This option is mandatory, has to refer to an existing AMI in the region, and has to comply with AMI ID format.
+
+	- *publicSshKey:* The administrator has to provide the name of the AWS keypair to be use to operate the instance supporting nodes.
+ This option is mandatory, and must refer to an existing AWS keypair in the region.
+
+	- *defaultVpcId:* This parameter can be filed with the the ID of the VPC to use to operate instance operating nodes.
+ If specified, this parameter has to refer to an existing VPC in the region and comply with the VPC ID format. 
+ If left blank, the connector the extension will trigger networking autoconfiguration.
+
+	- *defaultSubNetId:* The administrator can define which subnet has to be attached to the the instance supporting nodes.
+ If specified, this parameter has to refer to an existing subnet in the region affected to the specified VPC, and has to comply with the subnet ID format.
+ This parameter has to be filed only if the *defaultVpcId* is also completed.
+ Otherwise, this parameter has to be left blank to trigger networking autoconfiguration.
+
+	- *defaultSecurityGroup:* This parameter receives the ID of the security group to spawn instances into.
+ If this parameter does not meet the requirement regarding the providing the provided VPC and subnet, a new security group will be generated.
+ This parameter is mandatory, and has to comply with the format of the ID of the AWS security groups.
+
+	- *region:* The administrator specifies here the AWS region to allocate the cluster into.
+ This parameter is mandatory and has to be valued after the name of an AWS region.
```

Comment:
I have seen you used "be valued" as a synonym to "be assigned a value", I feel it strange as I think "be valued" means rather "be appreciated", but maybe I am wrong.

Reply:
Ok, let's switch to *configured*